site/themes/arangodb-docs-theme/layouts/partials/stylesheet.html

```diff
@@ -1,4 +1,7 @@
 {{ "<!-- https://github.com/filamentgroup/loadCSS/blob/master/README.md#how-to-use -->" | safeHTML }}
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet">
```

Comment:
If we load the only font we use viaGoogle, shall we delete `site/themes/arangodb-docs-theme/static/fonts`?

Reply:
Ah, we can just statically serve them if we already have that; will do